src/foam/nanos/logger/ILogger.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.nanos.logger;
+
+public interface ILogger {
+  public void start();
```

Comment:
oh sorry, start is not needed because of the nanoservice, it is needed because filelogger and stdoutlogger have start() functions, and this function needs to be called in nanologger.

like this

`  public void add(ILogger logger) {
    logger.start();
    delegates_.add(logger);
  }`